application/src/main/java/org/togetherjava/tjbot/features/basic/OofsAndLmaosStarboard.java

```diff
@@ -0,0 +1,27 @@
+package org.togetherjava.tjbot.features.basic;
+
+import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
+import net.dv8tion.jda.api.events.message.react.MessageReactionAddEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.jetbrains.annotations.NotNull;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.config.OofsAndLmaosConfig;
+import org.togetherjava.tjbot.features.EventReceiver;
+
+public class OofsAndLmaosStarboard extends ListenerAdapter implements EventReceiver {
```

Comment:
i dont like the `OofAndLmaos` everywhere. Id rename the whole feature to `Starboard` only (also for the config and everything)

Reply:
Done